helm/vitess/templates/vitess.yaml

```diff
@@ -3,6 +3,22 @@
 # create a single vttablet service
 {{ include "vttablet-service" $ }}
 ---
+{{ if $.Values.orchestrator.enabled }}
+
+# create orchestrator global services and StatefulSet
+{{ include "orchestrator" (tuple $.Values.orchestrator) }}
+---
+# create orchestrator config map
+{{ include "orchestrator-config" (tuple $.Values.orchestrator $.Release.Namespace) }}
+---
+# create a Service per StatefulSet replica
+{{ range $i := until (int $.Values.orchestrator.replicas) }}
+{{ include "orchestrator-statefulset-services" (tuple $.Values.orchestrator $i) }}
```

Comment:
Nit: change the template name to singular?

Reply:
Will do. I originally looped inside the template when I named it services, but decided to pull all that logic out inside the main loop.